File: Semester 4/Programming and Projection Enironments/mpp-proiect-java/Proiect_MPP/AirportModel/src/main/java/root/model/DTOFlight.java
package root.model;

import root.model.people.Client;
import root.model.people.Person;

import java.io.Serializable;
import java.util.List;

public class DTOFlight implements Serializable {
    private final Client client;
    private final List<Person> people;
    private final Flight flight;

    public DTOFlight(Client client, List<Person> people, Flight flight) {
        this.client = client;
        this.people = people;
        this.flight = flight;
    }

    public Client getClient() {
        return client;
    }

    public List<Person> getPeople() {
        return people;
    }

    public Flight getFlight() {
        return flight;
    }

    @Override
    public String toString() {
        return "DTOAirportFlight{" +
                "client=" + client +
                ", people=" + people +
                ", flight=" + flight +
                '}';
    }
}

File: Semester 4/Programming and Projection Enironments/mpp-proiect-csharp/Proiect_MPP/model/DTOFlight.cs
using model.people;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace model
{
    [Serializable]
    public class DTOFlight
    {
        public Client client { get; }
        public List<Person> people { get; }
        public Flight flight { get; }

        public DTOFlight(Client client, List<Person> people, Flight flight)
        {
            this.client = client;
            this.people = people;
            this.flight = flight;
        }

        public override string ToString()
        {
            return "DTOFlight{" +
                    "client=" + client +
                    ", people=" + people +
                    ", flight=" + flight +
                    '}';
        }
    }
}
